src/app.js

```diff
@@ -1,20 +1,32 @@
 const express = require('express');
 const bcrypt = require('bcrypt');
+const cron =  require('node-cron');
 require('./database.js')();
 
 const app = express();
 app.use(express.json());
 
+app.use((req, res, next) => {
+    res.setHeader("Access-Control-Allow-Origin", "*");
+    res.setHeader('Access-Control-Allow-Methods', 'DELETE');
+    res.header(
+      "Access-Control-Allow-Headers",
+      "Origin, X-Requested-With, Content-Type, Accept"
+    );
+    next();
+});
+
 //#region --- /notes ROUTES
 
 // CREATE new note
 app.post("/notes", async (req, res) => {
     try {
-        const { userid, title, content, url } = req.body;
-        const note = await createNote(userid, title, content, url);
+
+        const { userid, title, content, url, is_deleted, date} = req.body;
+        const note = await createNote(userid, title, content, url, is_deleted, date);
 
```

Comment:
Should we make these default values, rather than having to be specified in the API call? I think it would be best for adapting our existing code and tests.
```suggestion
        const { userid, title, content, url} = req.body;
        const note = await createNote(userid, title, content, url, 0, null);

```